paf-mvp-cmp/tests/model.test.ts

```diff
@@ -1,7 +1,29 @@
-import { Marketing, Model } from '../src/model';
+import { FieldSingle, Marketing, Model } from '../src/model';
+import { PreferencesData } from '@core/model/generated-model';
+import { isModuleDeclaration } from 'typescript';
 
 let model: Model;
 
+const signedPersonalized = {
+  data: Marketing.personalized,
+  version: '0.1',
+  source: {
+    domain: 'test.com',
+    signature: 'signature',
+    timestamp: 12345,
+  },
+};
+
+const signedStandard = {
+  data: Marketing.standard,
+  version: '0.1',
+  source: {
+    domain: 'test.com',
+    signature: 'signature',
+    timestamp: 12345,
+  },
+};
+
 describe('testing model', () => {
   beforeEach(() => {
     model = new Model();
```

Comment:
In the tests below, I think the test title is misleading: both mention "customized" fields, but the second tests "changeable" fields.
Can you please add comments or rename the test case?